static/app/views/projectsDashboard/resources.tsx

```diff
@@ -16,40 +16,33 @@ type Props = {
   organization: Organization;
 };
 
-class Resources extends Component<Props> {
-  componentDidMount() {
-    const {organization} = this.props;
+function Resources({organization}: Props) {
+  useEffect(() => {
     trackAnalyticsEvent({
```

Comment:
Should we add a `useEffectAnalytics`?

Reply:
That could be something we add as we get further down the line.